metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorPartitionService.java

```diff
@@ -105,26 +108,16 @@ public HiveConnectorPartitionService(
     public List<PartitionInfo> getPartitions(
         final ConnectorRequestContext requestContext,
         final QualifiedName tableName,
-        final PartitionListRequest partitionsRequest
-    ) {
-        try {
-            final List<Partition> partitions = getPartitions(tableName,
-                partitionsRequest.getFilter(), partitionsRequest.getPartitionNames(),
-                partitionsRequest.getSort(), partitionsRequest.getPageable());
-            final Table table = metacatHiveClient.getTableByName(tableName.getDatabaseName(), tableName.getTableName());
-            final TableInfo tableInfo = hiveMetacatConverters.toTableInfo(tableName, table);
-            final List<PartitionInfo> partitionInfos = new ArrayList<>();
-            for (Partition partition : partitions) {
-                partitionInfos.add(hiveMetacatConverters.toPartitionInfo(tableInfo, partition));
-            }
-            return partitionInfos;
-        } catch (NoSuchObjectException exception) {
-            throw new TableNotFoundException(tableName, exception);
-        } catch (MetaException | InvalidObjectException e) {
-            throw new InvalidMetaException("Invalid metadata for " + tableName, e);
-        } catch (TException e) {
-            throw new ConnectorException(String.format("Failed get partitions for hive table %s", tableName), e);
+        final PartitionListRequest partitionsRequest,
+        final TableInfo tableInfo) {
```

Comment:
I thought we introduced the TableInfo because it was not being called in getPartitions. I see that we used to call. Do we still need to make the wide-spread interface change?